recipes/pyemf/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyemf" %}
+{% set version = "2.1.2-alpha" %}
+{% set vers_list = version.split('-') %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ vers_list[0] }}
+
+source:
+  url: https://github.com/mikespub-org/bjodah-pyemf/archive/v{{ version }}.tar.gz
```

Comment:
We generally prefer to target the original repo instead of a fork. Is that one completely dead?

Reply:
Upstream repo has been updated so I switched to this one instead of a fork